experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -393,10 +392,17 @@ function Interfacer.add_coupler_fields!(coupler_field_names, atmos_sim::ClimaAtm
     push!(coupler_field_names, atmos_coupler_fields...)
 end
 
+"""
+    Interfacer.close_output_writers!(sim::ClimaAtmosSimulation)
+
+Close all output writers used by the atmos simulation.
+"""
+Interfacer.close_output_writers!(sim::ClimaAtmosSimulation) =
```

Comment:
I don't think this should be a mutating function. In Julia, operating on the enviroment (e.g., IO) is not considered mutating. (It's `close` not `close!`)

Reply:
I didn't know that, thank you!